src/Svg.CodeGen.Skia/SkiaCSharpModelExtensions.cs

```diff
@@ -1063,6 +1063,26 @@ public static void ToSKImageFilter(this SKImageFilter? imageFilter, SkiaCSharpCo
                 sb.AppendLine($"{indent}{counter.PaintVarName}{counterPaint}?.Dispose();");
                 return;
             }
+            case ShaderImageFilter shaderImageFilter:
+            {
+                if (shaderImageFilter.Shader is null)
+                {
+                    sb.AppendLine($"{indent}var {counter.ImageFilterVarName}{counterImageFilter} = default(SKImageFilter);");
+                    return;
+                }
+
+                var counterShader = ++counter.Shader;
+                shaderImageFilter.Shader.ToSKShader(counter, sb, indent);
+
+                sb.Append($"{indent}var {counter.ImageFilterVarName}{counterImageFilter} = ");
+                sb.AppendLine($"SKImageFilter.CreateShader(");
+                sb.AppendLine($"{indent}    {counter.ShaderVarName}{counterShader},");
+                sb.AppendLine($"{indent}    {shaderImageFilter.Dither.ToBoolString()},");
+                sb.AppendLine($"{indent}    {shaderImageFilter.Clip?.ToCropRect() ?? "null"});");
+
+                sb.AppendLine($"{indent}{counter.ShaderVarName}{counterShader}?.Dispose();");
+                return;
+            }
```

Comment:
Not sure if I did this right... I don't think this code path will be hit currently as no current API actually uses shader image filters, but this is for when we do.